test/e2e/density.go

```diff
@@ -79,7 +82,7 @@ var _ = Describe("Density", func() {
 		// Verify latency metrics
 		// TODO: Update threshold to 1s once we reach this goal
 		// TODO: We should reset metrics before the test. Currently previous tests influence latency metrics.
-		highLatencyRequests, err := HighLatencyRequests(c, 10*time.Second, util.NewStringSet("events"))
+		highLatencyRequests, err := HighLatencyRequests(c, 1*time.Second, util.NewStringSet("events"))
```

Comment:
Let's not change this to 1 second yet. Otherwise it will never pass and it will be harder to spot regression.